src/routes/safe/components/Apps/hooks/useAppList.ts

```diff
@@ -96,11 +99,28 @@ const useAppList = (): UseAppListReturnType => {
     [appList],
   )
 
+  const onAppRemoved: onAppRemovedHandler = useCallback(
+    (appId) => {
+      // update in-memory list
+      const appListCopy = [...appList]
```

Comment:
nitpick: this could've been a one-liner using .filter

Reply:
reimplemented